docs/product/base/grid.html

```diff
@@ -0,0 +1,808 @@
+    <div class="stacks-preview">
+{% highlight html %}
+<div class="d-grid grid__4">
+    <div class="grid--col2 grid--row-start2 grid--row-end4">…</div>
+    <div class="grid--col2">…</div>
+    <div class="grid--col2">…</div>
+    <div class="grid--col2">…</div>
+    <div class="grid--col2">…</div>
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example ff-mono ws-nowrap">
+            <div class="d-grid grid__4 gg16">
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2 grid--row-start2 grid--row-end4">
+                    <div>.grid--col2</div>
+                    <div>.grid--row-start2</div>
+                    <div>.grid--row-end4</div>
+                </div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="stacks-section">
+    {% header h2 | Alignment %}
+    <p class="stacks-copy">On the grid container, you can apply <code class="stacks-code">align-items</code> to the y axis and <code class="stacks-code">justify-items</code> to the x axis. On individual items, you can apply <code class="stacks-code">align-self</code> on the y axis, and <code class="stacks-code">justify-self</code> on the x axis.</p>
+
+    {% header h3 | Alignment classes %}
+    <div class="overflow-x-auto mb48">
+        <table class="wmn4 s-table s-table__bx-simple">
+            <thead>
+                <tr>
+                    <th class="s-table--cell3" scope="col">Class</th>
+                    <th class="s-table--cell3" scope="col">Output</th>
+                    <th scope="col">Definition</th>
+                    <th class="s-table--cell2 ta-center" scope="col"><a class="s-link s-link__inherit" href="{{ "/product/guidelines/conditional-classes#responsive" | relative_url }}">Responsive?</a></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for item in site.data.product.css-grid.alignment %}
+                    <tr class="fs-caption">
+                        <th scope="row"><code class="stacks-code">.{{ item.class }}</code></th>
+                        <td><code class="stacks-code bg-white">{{ item.output }}</code></td>
+                        <td>{{ item.define }}</td>
+                        <td class="ta-center">
+                            {% if item.responsive %}
+                                {% icon Checkmark | fc-green-500 %}
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+
+    {% header h3 | Alignment examples %}
```

Comment:
Thanks for adding these classes. There's a ton of options in the table above and I like how we're trying to visualize some of it. This is helpful and I wish we'd do it on our flexbox page too.

However this example is a bit confusing. We're trying to visualize _some_ of the options above, but I'm not quite sure what's going on with these six boxes and why they're laying out they way they are.

I'm not sure what the answers is... maybe a bunch of tiny examples like on [CSS Tricks](https://css-tricks.com/snippets/css/complete-guide-grid/)? Or maybe no example?

Reply:
I like having just a small example so folks have it in their head what these classes do, but I draw a line at explaining every part of the CSS spec. I think this strikes the balance. Lemme know if I'm completely wrong 😛

Reply:
Yea, I get that. I also don't think we should draw a ton of tiny diagrams. But I'm not sure if this example is helping much, maybe worth killing it and going back to words only?